src/languages/es.js

```diff
@@ -707,6 +707,14 @@ export default {
             setDefaultFailure: 'No se ha podido configurar el método de pago.',
         },
         addBankAccountFailure: 'Ocurrió un error inesperado al intentar añadir la cuenta bancaria. Inténtalo de nuevo.',
+        cardDetails: {
```

Comment:
Has Expensify confirmed these translations?

Reply:
In progress ⏳